src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -2370,11 +2445,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
-    Friend Partial Class BoundByRefArgumentWithCopyBack
+    Partial Friend Class BoundByRefArgumentWithCopyBack
         Protected Overrides Function ExpressionKind() As OperationKind
             Return OperationKind.None
         End Function
 
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation)
+            Get
+                Return ImmutableArray.Create(Of IOperation)(Me.OriginalArgument, Me.InConversion, Me.InPlaceholder, Me.OutConversion, Me.OutPlaceholder)
```

Comment:
>Me.InConversion, Me.InPlaceholder, Me.OutConversion, Me.OutPlaceholder [](start = 81, length = 70)

I think we shouldn't expose these nodes for now. Placeholders are going to be duplicated in conversions.